IndicoV2.Abstractions/Models/IModelClient.cs

```diff
@@ -15,13 +15,6 @@ public interface IModelClient
         /// <returns></returns>
         Task<IModelGroup> GetGroup(int modelGroupId, CancellationToken cancellationToken);
 
-        /// <summary>
-        /// Loads Model
-        /// </summary>
-        /// <param name="modelId"><seealso cref="IModel"/>'s Id</param>
-        /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asynchronous operations.</param>
-        /// <returns>Status</returns>
-        Task<string> LoadModel(int modelId, CancellationToken cancellationToken);
```

Comment:
I know this is more work, but I would like to mark this as @ deprecated and make the underlying call return a no-op/success by default. We don't want to delete API calls without first marking them deprecated so users can migrate.